keyboards/dtd/ldmlKeyboard3.dtd

```diff
@@ -0,0 +1,158 @@
+<!--
+Copyright © 1991-2022 Unicode, Inc.
+For terms of use, see http://www.unicode.org/copyright.html
+SPDX-License-Identifier: Unicode-DFS-2016
+CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
+-->
+<!-- Important Note:
+
+The CLDR Keyboard Workgroup is currently developing major changes to the
+CLDR keyboard specification.
+
+This DTD is a work in progress.
+
+Please see CLDR-15034 for the latest information. -->
+
+<!ELEMENT keyboard ( version, info?, names, settings?, import*,
+	vkeyMaps*, displayMap*, keyMap+, layerMaps,
```

Comment:
Do we need multiple `keyMap`s?

Reply:
the reason there might be more would be if there is an alternate of some kind.   But, with `modifiers=` removed, there is no possibility for an alternate.  CLDR sometimes has `alt=…` as an attribute, to allow alternates.  

So, i think it should just be `keyMap,` - one exactly is required.

Reply:
```suggestion
	vkeyMaps*, displayMap*, keyMap, layerMaps,
```